balancer-js/src/modules/data/block-number/index.ts

```diff
@@ -21,36 +20,32 @@ const fetchBlockByTime = async (
   endpoint: string,
   timestamp: string
 ): Promise<number> => {
-  console.time(`fetching blocks ${timestamp}`);
   const payload = {
     query: query(timestamp),
   };
 
-  const response = await axios.post(endpoint, payload);
-  console.timeEnd(`fetching blocks ${timestamp}`);
+  const response = await fetch(endpoint, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(payload),
+  });
 
   const {
     data: { blocks },
-  } = response.data as BlockNumberResponse;
+  } = (await response.json()) as BlockNumberResponse;
 
   return parseInt(blocks[0].number);
 };
 
 export class BlockNumberRepository implements Findable<number> {
-  blocks: { [ts: string]: Promise<number> } = {};
-
   constructor(private endpoint: string) {}
 
   async find(from: string): Promise<number | undefined> {
     if (from == 'dayAgo') {
       const dayAgo = `${Math.floor(Date.now() / 1000) - 86400}`;
-      if (!this.blocks[dayAgo]) {
-        this.blocks = {
-          ...this.blocks,
-          [dayAgo]: fetchBlockByTime(this.endpoint, dayAgo),
-        };
-      }
-      return this.blocks[dayAgo];
+      return fetchBlockByTime(this.endpoint, dayAgo);
     }
   }
 
```

Comment:
seems like this file got mixed up after rebase. should be with axios, so all modifications to it can be rolled back